opendocdb-cts/internal/mongosh/fixture_test.go

```diff
@@ -162,9 +177,75 @@ func TestConvertFixtures(t *testing.T) { //nolint:revive // exceeds number of li
 		},
 	} {
 		t.Run(name, func(t *testing.T) {
-			actual, err := ConvertFixtures(tc.fixtures)
+			t.Parallel()
+			if tc.skip != "" {
+				t.Skip(tc.skip)
+			}
+
+			ctx := context.Background()
+			l := slogt.New(t)
+			dbName := strings.ReplaceAll(t.Name(), "/", "-")
+
+			actualJS, err := ConvertFixtures(tc.fixtures)
+			require.NoError(t, err)
+			assert.Equal(t, unindent(tc.expected)+"\n", actualJS)
+
+			mountPath := filepath.Join(mountDir, name+".js")
+			err = os.WriteFile(mountPath, []byte(actualJS), 0o666)
 			require.NoError(t, err)
-			assert.Equal(t, unindent(tc.expected)+"\n", actual)
+
+			conn, err := wireclient.Connect(ctx, "mongodb://127.0.0.1:27001/", l)
+			require.NoError(t, err)
+
+			t.Cleanup(func() {
+				_ = os.Remove(mountPath)
```

Comment:
`_ = os.Remove(mountPath)` should be done right after successful `os.WriteFile`, not after `wireclient.Connect`.